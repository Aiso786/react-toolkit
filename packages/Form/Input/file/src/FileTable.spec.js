import React from 'react';
import { shallow } from 'enzyme';
import FileTable from './FileTable';

describe('<File.FileTable>', () => {
  it('renders File.FileTable correctly', () => {
    const wrapper = shallow(
      <FileTable
        errors={[{ file: { name: 'youhou.txt', size: 2 } }]}
        values={[{ file: {}, id: 'id' }]}
        onClick={selectedId => console.log(selectedId)}
      />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
