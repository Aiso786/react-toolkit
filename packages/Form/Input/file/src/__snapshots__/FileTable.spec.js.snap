// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<File.FileTable> renders File.FileTable correctly 1`] = `
<div
  className="custom-table-file af-file-table"
>
  <div
    style={
      Object {
        "color": "red",
      }
    }
  >
    <span>
      Le chargement de certains fichiers à échoué: 
    </span>
    <ul>
      <li
        key="[object Object]"
      >
         
        youhou.txt
         (
        2
        )
         
      </li>
    </ul>
  </div>
  <table
    className="table table-striped table-bordered"
  >
    <tbody>
      <FileLine
        classModifier={null}
        className="custom-line-file af-file-line"
        disabled={false}
        file={Object {}}
        id="id"
        key="id"
        onClick={[Function]}
      />
    </tbody>
  </table>
</div>
`;
